temporal/logging.go

```diff
@@ -36,16 +37,18 @@ func ExtractLoggerMetadata(header *common.Header) (map[string]string, error) {
 
 	out := make(map[string]string)
 	for _, field := range fields {
-		out[field.Key] = field.Value
+		out[field.Key] = field.Value.(string)
```

Comment:
Same here - I don't think that `Value` should be `any`. This cast will cause a panic if the underlying value isn't really a string.

Reply:
I changed the type of `Value`, this is resolved